app/src/main/java/com/codesoom/assignment/TodoHttpHandler.java

```diff
@@ -0,0 +1,175 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.controller.TodoHttpController;
+import com.codesoom.assignment.models.HttpStatus;
+import com.codesoom.assignment.models.Todo;
+import com.codesoom.assignment.validate.TodoValidator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class TodoHttpHandler implements HttpHandler {
+    private final TodoHttpController todoHttpController;
+    private final TodoValidator todoValidator;
+    private final ObjectMapper objectMapper;
+    public TodoHttpHandler() {
+        this.todoHttpController = new TodoHttpController();
+        this.todoValidator = new TodoValidator();
+        this.objectMapper = new ObjectMapper();
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) {
+        try {
+            String requestMethod = exchange.getRequestMethod();
+            String path = exchange.getRequestURI().getPath().substring(1);
```

Comment:
가급적이면 변경할 일이 없는 변수들은 `final`을 붙여서 코드를 읽는 사람이 처음부터 "아 이건 바꾸지 않는 값이구나!"라고 생각할 수 있게 해주세요.

```suggestion
            final String requestMethod = exchange.getRequestMethod();
            final String path = exchange.getRequestURI().getPath().substring(1);
```